### DISCUSSION TITLE
How can I see full process command line?

### DISCUSSION THREAD

**Discussion by Gorthog** [ClementTsang/bottom#520] [Q&A]
I have some very lengthy process command lines which are truncated:
![image](https://user-images.githubusercontent.com/15230040/110239356-3566b700-7f4f-11eb-88ac-785341771632.png)

Is there a way to see the full command line of the process?

for example in [procs](https://github.com/dalance/procs) i can scroll horizontally:
![image](https://user-images.githubusercontent.com/15230040/110239449-91c9d680-7f4f-11eb-9ffa-4a513a915e18.png)
![image](https://user-images.githubusercontent.com/15230040/110239456-9c846b80-7f4f-11eb-83b5-ba91b18dfb3b.png)

**Comment by ClementTsang** [comment]
Unfortunately, there isn't a way to do so for now.  It's part of a shortcoming with how I initially designed the process widget, and where I placed the column.

I do plan on addressing this in the future in some way.

EDIT: What I'll probably(?) do in the future is separating out the command and process columns.  Currently they're tied together, I'll probably move the command column as an entirely separate thing and put it as the last column, and add the ability to scroll left/right.

**Comment by guija** [comment]
I think this would be still a good feature to have. At the moment I have to leave bottom and then use another tools to see the entire command (such as `ps`) which I would like to avoid to not switch between too many tools.

**Reply by ClementTsang** [reply]
Just as an update, there's https://github.com/ClementTsang/bottom/issues/1129 to work on this.
